### ISSUE TITLE
unable to install via pip

### ISSUE THREAD

**Issue by extraymond** [tekulvw/bvh-converter#2]
```
Collecting bvh-converter
  Using cached bvh-converter-1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-xbbyyra5/bvh-converter/setup.py", line 28, in <module>
        install_requires=get_requirements(),
      File "/tmp/pip-build-xbbyyra5/bvh-converter/setup.py", line 5, in get_requirements
        with open('requirements.txt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-xbbyyra5/bvh-converter/
```

Tried with both python 3.5 and 3.6.

**Comment by extraymond** [comment]
[possible fix on stackoverflow](https://stackoverflow.com/a/43748965)

**Comment by tekulvw** [comment]
Fixed with #3
